Code/Framework/AzQtComponents/AzQtComponents/Components/StyleManager.cpp

```diff
@@ -169,6 +169,10 @@ namespace AzQtComponents
         initializeSearchPaths(application, engineRootPath);
         initializeFonts();
 
+        QFont defaultFont("Open Sans");
+        defaultFont.setPixelSize(12);
```

Comment:
So this mean we'll always have a uniform 12px font regardless of screen scaling factor? I'm a bit concerned about legibility on high pixel density devices, but perhaps Qt is actually scaling everything up for us even with a pixel size?

Reply:
No, if a pixel value is set the painter will actually scale it according to the scaling setting. For some reason though, it doesn't when a default font isn't set (pixelSize() returns -1).